src/main/java/uk/gov/companieshouse/account/validator/model/FileInfo.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.account.validator.model;
```

Comment:
This should be in the [api sdk](https://github.com/companieshouse/api-sdk-java).